Bulls_Cows.py

```diff
@@ -0,0 +1,78 @@
+import random
+
+def generate_list():
+    lst = [i for i in range(10)]
+    lst = random.sample(lst, 4)
+    return lst
+
+def user_parse():
+        while True:
+            try:
+                user_try = list(input("Enter four number: "))
+                if len(user_try) != 4:
+                    raise Exception
+                if len(user_try) == 4:
+                    for i in user_try:
+                        int(i)
+                    return user_try
+                else:
+                    return user_try
+            except:
+                print('You have to enter four number, for instance 1234 ')
+                continue
+
+def comparison(target_list, user_try):
+    for_target = list(target_list)
+    bulls = 0
+    cows = 0
+    for index in range(4):
+        if target_list[index] == int(user_try[index]):
```

Comment:
Why don't you convert the `user_try` elements to int `user_parse`? You are using there `int` function but don't assign it anywhere.